src/javascript/app/pages/cashier/payment_agent_withdraw.js

```diff
@@ -71,8 +71,23 @@ const PaymentAgentWithdraw = (() => {
 
             const form_id = `#${$(view_ids.form).find('form').attr('id')}`;
             const $form   = $(form_id);
-            const min     = getPaWithdrawalLimit(currency, 'min');
-            const max     = getPaWithdrawalLimit(currency, 'max');
+
+            const min  = () => {
+                const selected_val = $ddl_agents.val() || $txt_agents.val();
+                if (selected_val){
+                    const selected_pa = pa_list.find(pa => pa.paymentagent_loginid === selected_val);
+                    if (selected_pa) return selected_pa.min_withdrawal;
+                }
+                return getPaWithdrawalLimit(currency, 'min');
+            };
+            const max  = () => {
+                const selected_val = $ddl_agents.val() || $txt_agents.val();
+                if (selected_val){
+                    const selected_pa = pa_list.find(pa => pa.paymentagent_loginid === selected_val);
+                    if (selected_pa) return selected_pa.max_withdrawal;
+                }
+                return getPaWithdrawalLimit(currency, 'max');
+            };
```

Comment:
I think you could combine these 2 functions to 1, or move the function body to a new shared function called in min and max since the logic is more or less the same

Reply:
Alright 👍